device-types/Mellanox/qm8700.yml

```diff
@@ -0,0 +1,100 @@
+---
+manufacturer: Mellanox
+model: QM8700
+slug: mellanox-qm8700
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: ps1
+    type: iec-60320-c14
+    maximum_draw: 1100
+  - name: psu2
+    type: iec-60320-c14
+    maximum_draw: 1100
```

Comment:
These appear to be modular PSUs. Please convert these to `module-bays` and create the associated PSU modules within the module-types folder if they do not already exist